gradle/libs.versions.toml

```diff
@@ -1,95 +1,98 @@
 [versions]
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
```

Comment:
Some version bumps, but mostly reorganization to the dank hyphenated format that makes things dot-qualified in the build files.